### PR TITLE
switch to tiered consolidation by default

### PR DESCRIPTION
**Pull request by gnusi**
Please note that for legal reasons we require you to sign the 
[Contributor Agreement](https://www.arangodb.com/documents/cla.pdf)
before we can accept your pull requests.

### CODE REVIEW

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/4432/

File: arangod/IResearch/IResearchViewMeta.cpp [hunk 1]
```diff
@@ -158,14 +159,30 @@ arangodb::iresearch::IResearchViewMeta::ConsolidationPolicy createConsolidationP
...
   }
 
+  {
+    // optional double
+    static const std::string fieldName("minScore");
+
+    if (slice.hasKey(fieldName)) {
+      auto field = slice.get(fieldName);
+
+      if (!field.isNumber<size_t>()) {
```
Comment (jsteemann) [inline arangod/IResearch/IResearchViewMeta.cpp]: is it intentional that here you are checking for a `size_t` and use a `double` for the following `getNumber` call?

Comment (gnusi) [reply arangod/IResearch/IResearchViewMeta.cpp]: well spotted, copy paste error

**Comment by gnusi** [comment]
https://jenkins.arangodb.biz/job/arangodb-matrix-pr-linux/5961/

**Comment by gnusi** [comment]
https://jenkins.arangodb.biz/job/arangodb-matrix-pr-linux/5962/

**Pull request merged**